src/experiments/buttonDesign/script.js

```diff
@@ -0,0 +1,8 @@
+/* @flow */
+
+import { html } from './html';
+import { script } from './treatments/divideLogoAnimation';
```

Comment:
Is `divideLogoAnimation` just one treatment for button designs experiment?

Reply:
@mnicpt Yes it is